### ISSUE TITLE
Command "vscode-aura.testAura" not found

### ISSUE THREAD

**Issue by SilverCrovv** [obfuscatedgenerated/vscode-aura#10]
I guess the issue in the Electron. Could you please take a look?

**Comment by obfuscatedgenerated** [comment]
> I guess the issue in the Electron

Yeah, that's right. I'm pushing an update shortly. It's a real pain to update the abi each time a new vscode build is published so this project has gone a little stale. Thanks for checking it out!

**Comment by obfuscatedgenerated** [comment]
Actually, I'm having some trouble updating the Electron version, and resolving it is going to take quite a bit of work. I think I'm going to archive this project and discontinue the extension. It's just too much work to keep up to date. Thank you for trying it out anyway, sorry for any inconvenience.
